common/neon_rpc/server.py

```diff
@@ -67,6 +67,7 @@ def _create_env(self) -> dict[str, Any]:
             COMMITMENT="recent",
             EVM_LOADER=str(NeonProg.ID),
             NEON_DB_CLICKHOUSE_URLS=";".join(self._cfg.ch_dsn_list),
+            TRACER_DB_TYPE="clickhouse",
```

Comment:
Set up `TRACER_DB_TYPE` to `clickhouse` or `none` depends on `self._cfg.ch_dsn_list` value.